templates/home/_fsf_python.html

```diff
@@ -2,16 +2,18 @@
   <div class='col-6'>
     <h4>Why are snaps good for Python projects?</h4>
     <ul>
-      <li>Snaps are easy to discover and install. Millions of users can browse and install snaps graphically in the Snap Store or from the command-line.</li>
-      <li>Snaps install and run the same across Linux. They bundle the exact version of Python required and all of your app’s dependencies, be they Python modules or system libraries.</li>
-      <li>Snaps automatically update to the latest version. Four times a day, users’ systems will check for new versions and upgrade in the background.</li>
-      <li>Upgrades are not disruptive. Because upgrades are not in-place, users can keep your app open as it’s upgraded in the background.</li>
-      <li>Upgrades are safe. If your app fails to upgrade, users automatically roll back to the previous revision.</li>
+      <li>Easy to discover and install by millions using the Snap Store or command-line every day</li>
+      <li>Automatically updated to the latest stable version of your app</li>
+      <li>Revert to previous version if an update fails, preserving data</li>
+      <li>Isolation ensures no conflicts between applications</li>
+      <li>Identical behaviour across Linux distributions, even with C library dependencies</li>
     </ul>
```

Comment:
This list is quite hard to read, because it’s unusual for every point in a list to have different grammar: adjective phrase, past-tense verb phrase, imperative verb phrase, clause, noun phrase. The second item in particular looks incomplete: What is updated? When? Perhaps they could all be clauses: “The Snap Store makes your app easy to discover and install…”, “Automatic updates migrate your users to the latest…”. “Isolation prevents conflicts…” etc

Reply:
Can you please make a suggestion for content?

Reply:
To expand on my suggestion above:
```html
<li>The Snap Store makes your app easy to discover and install for millions of users, graphically or at the command line</li>
<li>Automatic updates migrate your users to the latest stable version</li>
<li>If an update fails, the previous version is reverted, preserving data</li>
<li>Isolation prevents conflicts between applications</li>
<li>Identical behaviour across Linux distributions, even with C library dependencies</li>
```